Tasks/DownloadSecureFile/task.json

```diff
@@ -31,6 +31,14 @@
             "defaultValue": "",
             "required": true,
             "helpMarkDown": "Select the secure file to download to a temporary location on the agent. The file will be cleaned up after the build or release."
+        },
+        {
+            "name": "secureFilePath",
```

Comment:
A better name would be "secureFileDownloadDirectory" or "secureFileDownloadPath"